test/mocks/statement.js

```diff
@@ -11,7 +11,11 @@ const mockStatement1 = {
   sbi: 123456789,
   frn: 1234567890,
   email: 'farmer1@farm.com',
-  received: new Date(2022, 7, 5, 15, 30, 10, 120)
+  received: new Date(2022, 7, 5, 15, 30, 10, 120),
+  schemeName: 'Sustainable Farming Incentive',
+  schemeShortName: 'SFI',
+  schemeYear: '2022',
+  schemeFrequency: 'Quarterly'
 }
 
 const mockStatement2 = {
```

Comment:
Would using object spread operator make this more 1) readable and 2) ensure consistency w/ object keys?